allensdk/brain_observatory/ecephys/stimulus_table/output_validation.py

```diff
@@ -14,8 +14,9 @@ def validate_epoch_durations(table, start_key="Start", end_key="End"):
         )
     if min_duration < 0:
         print(table.loc[min_duration_index - 5 : min_duration_index + 5, :])
-        raise ValueError(
-            f"there is an epoch with negative duration (index: {min_duration_index}"
+        warnings.warn(
```

Comment:
I don't agree with this change. A negative duration epoch calls into question the stimulus timing for this session.